sdk/firstload/package.json

```diff
@@ -28,7 +28,7 @@
 		"build": "yarn typegen && vite build && yarn build:umd",
 		"build:umd": "cp dist/index.umd.cjs dist/index.umd.js",
 		"build:watch": "yarn build --watch",
-		"dev": "vite dev",
+		"dev": "",
```

Comment:
why this change?

Reply:
I shouldn't have checked that in. 

We can't run both `yarn dev:frontend` and `yarn e2e:nextjs` at the same time, because this `vite dev` job fights over a port. I've been manually disabling the job to get both running. The better solution is probably to not run `dev` for `firstload` when running `yarn e2e:nextjs`.

Reply:
Fixed!